etl/src/main/java/org/pih/hivmigration/etl/sql/TreatmentObsMigrator.groovy

```diff
@@ -753,6 +750,7 @@ class TreatmentObsMigrator extends ObsMigrator {
                             obs_group_id
                      FROM hivmigration_tmp_arv_regimen) arv
             LEFT JOIN tmp_obs o ON o.obs_group_id = arv.obs_group_id
+                AND o.concept_uuid = concept_uuid_from_mapping('PIH', '1282')
```

Comment:
This query says "make a temp table of coded mappings for non-coded values. Then join on the existing obs to filter down to just the groups which don't already have a coded obs value." The problem was that it wasn't checking "whether there's already a coded obs value in the group," it was checking "whether there's anything in the group," which there always ways.

Reply:
Just to clarify, Is this for the issue with the art start date not getting migrated over if a regimen wasn't specified from my last comment on the ticket... ?  This changes seems unrelated but maybe I'm missing something?

Reply:
Since this ART entry wasn't getting added, "Autre" was getting filled in for the coded medication. The date seems to be working, at least in the follow-up status section. It's even working for non-codable other values:

![Screenshot from 2021-03-19 11-57-37](https://user-images.githubusercontent.com/1031876/111829777-5e217180-88aa-11eb-9f80-e83d863d01ce.png)